script.js

```diff
@@ -1,18 +1,21 @@
 
 //start the game 
 let PlayerSelection = 'rock';
+let possibleChoices = ['rock', 'paper', 'scissors'];
+
     //computer chooses between rock, paper, scissors
 function getComputerChoice() {
+    debugger;
     const getChoice = Math.floor(Math.random() * possibleChoices.length)
-        if (getChoice =0){
```

Comment:
Looks like you switched up your "setting a variable" amount of = and your "comparing objects" amount of =
set a variable: always use 1 `=`
comparing objects: can use 2 or 3 `=`